src/service/url-replacements-impl.js

```diff
@@ -753,6 +193,7 @@ export function installUrlReplacementsServiceForDoc(ampdoc) {
  * @param {*} varSource
  */
 export function installUrlReplacementsForEmbed(ampdoc, embedWin, varSource) {
+  installVarSubstitutionForEmbed(ampdoc, embedWin, varSource);
   installServiceInEmbedScope(embedWin, 'url-replace',
-      new UrlReplacements(ampdoc, varSource));
+      new UrlReplacements(ampdoc));
```

Comment:
What do you mean service for doc is installed elsewhere, you mean `runtime.js` call? 

It would be a noop if we call it again here, but I don't understand why would we call it here since we haven't been installing it here before this change?